inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd

```diff
@@ -478,20 +483,29 @@ study_data_cleaned <- study_data_cleaned %>%
                                                  "Unvaccinated")
   )
 
+# explicitly replace NA of a factor
+study_data_cleaned <- study_data_cleaned %>%
+  mutate(vaccine_routine = fct_explicit_na(vaccine_routine, na_level = "No answer"))
 
 
 
 ######## weighting ------------------------------------------------------------
 
-study_data_cleaned <- add_weights(study_data_cleaned, population_data_age, age_group, sex)
+study_data_cleaned <- add_weights(x = study_data_cleaned, 
+                                  p = population_data_age, 
+                                  age_group, sex)
 
 # drop unwanted rows  ---------------------------------------------------------
 
+# store the cases that you drop so you can describe them (e.g. non-consenting)
+dropped <- study_data_cleaned %>% 
```

Comment:
Wait... wasn't this done earlier?

Reply:
nope. Dropping non consents happens in the line after. The only previous back up is of study_data_raw